Source/Scene/BoxEmitter.js

```diff
@@ -0,0 +1,45 @@
+/*global define*/
+define([
```

Comment:
My gut was that this would just be a primitive - which almost cuts the code/doc/tests in half.  If we want to put this in CZML later then we may want the Cesium `Property` system with the entity layer.

We plan to streamline the entity API eventually.  Perhaps it would be best to park the entity API for this in a branch until we have that figured out?  It will be much less to maintain.